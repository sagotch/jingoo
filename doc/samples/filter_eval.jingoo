{% set x1 = "{% set msg = 'Hello, world!' %}{% set src = 'From eval!' %}{{ msg }} {{ src }}" -%}
{% set x2 = "{% set msg = 'Hello, world!' %}{% set src = 'From eval!' %}{{ msg + ' ' + src }}" -%}
{% set x3 = "{% set msg = 'Hello, world!' %}{{ msg }} " -%}

<p>As in OCaml, only the last expression is returned.</p>
<p>{{ eval(x1) }}</p>
<p>{{ eval(x2) }}</p>
<p>As a consequence, you must be careful with trailing whitespaces.</p>
<p>{{ eval(x3) }}</p>
