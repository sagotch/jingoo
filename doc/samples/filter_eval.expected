<p>As in OCaml, only the last expression is returned.</p>
<p>From eval!</p>
<p>Hello, world! From eval!</p>
<p>As a consequence, you must be careful with trailing whitespaces.</p>
<p> </p>
